.evergreen/config.yml

```diff
@@ -251,7 +251,7 @@ functions:
         working_dir: src/go.mongodb.org/mongo-driver
         script: |
           ${PREPARE_SHELL}
-          ${BUILD_ENV|} make ${targets} BUILD_TAGS="-tags cse,gssapi"
+          ${BUILD_ENV|} make ${targets} BUILD_TAGS="-tags \"cse gssapi\""
```

Comment:
We need to go back to the legacy way of specifying multiple tags delimited by spaces rather than commas because these will now be used on with Go versions < 1.13. The escaped quotes are needed so the output because otherwise the `make` command interprets the second tag as a `make` target.